libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -45,14 +45,31 @@ func setupIPv6BridgeNetFiltering(config *networkConfiguration, _ *bridgeInterfac
 	return nil
 }
 
+func LoadBridgeModules() error {
+	moduleMap := map[string]string {
+		"arp_probe":                "bridge",
```

Comment:
With whatever Debian kernel I was using, I found that loading `bridge` doesn't create anything in `/proc/sys/net`. However, creating a bridge loads module `bridge` automatically.

That means the module will always be loaded before `enableBridgeNetFiltering` is called (because at-least `docker0` will have been created). Also, modprobe-ing `br_netfilter` loads `bridge` as a dependency.

So, I think we should drop the check for `arp_probe` and not try to `modprobe bridge` - as it'll happen each time a network is created on hosts where `arp_probe` isn't created (and it'll simplify the code a little).

Reply:
Agree, changed logic to just load `br_netfilter`